spring-ai-docs/src/main/antora/modules/ROOT/pages/api/vectordbs/chroma.adoc

```diff
@@ -79,7 +79,7 @@ spring.ai.vectorstore.chroma.collection-name=<your collection name>
 # Chroma Vector Store configuration properties
 
 # OpenAI API key if the OpenAI auto-configuration is used.
-spring.ai.openai.api.key=<OpenAI Api-key>
+spring.ai.openai.api-key=<OpenAI Api-key>
```

Comment:
could you also fix the same typo in other documents ? 

<img width="642" alt="image" src="https://github.com/user-attachments/assets/a791e365-bf56-4ef0-8a70-bd9933f7a816">